terra/templates/terra/fund.html

```diff
@@ -46,7 +46,7 @@ <h3>{{fund}}</h3>
         <tbody>
             {% for employee in employees %}
             <tr>
-                <td><a href="/employee/{{employee.pk}}/">{{employee.user.last_name}}, {{employee.user.first_name}}</a></td>
+                <td><a href="/employee/{{employee.pk}}/">{{employee.user.last_name}}, {{employee.user.first_name}}</a> {{employee.get_type_display}}</td>
```

Comment:
@aprigge Minor suggestion: This might look better with parentheses around the employee type, like this:
`Grappone, Todd (Unit Head)`
`Prigge, Ashton (Librarian)`

If so, the change should be made here (`fund.html`) and in `unit.html`.  If not, this can be merged as is.

What do you think?

Reply:
@akohler and @joshuago78 I made the changes Josh had suggested above. Thanks